guide/daily-spending-wallet/send-fees.md

```diff
@@ -0,0 +1,205 @@
+   retina = "/assets/images/guide/daily-spending-wallet/sending/send-fees/fee-options@2x.png"
+   modalImage = "/assets/images/guide/daily-spending-wallet/sending/send-fees/fee-options-big.png"
+   alt-text = "Screen showing fee options for an onchain transaction"
+   caption = "A simple framework for on-chain fees is the urgency of the transaction."
+   width = 250
+   height = 541
+   layout = "float-right-desktop -background -shadow"
+%}
+
+Generally, it is best to dynamically calculate a reasonable fee range and provide a few simple options based on how urgent the transaction is for the user. Total cost and confirmation time are often the primary decision points.
+
+</div>
+
+#### High fee situations
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/daily-spending-wallet/sending/send-fees/fee-options-high.png"
+   retina = "/assets/images/guide/daily-spending-wallet/sending/send-fees/fee-options-high@2x.png"
+   alt-text = "Screen showing a fee picker for an onchain transaction with high fee options marked in red"
+   width = 250
+   height = 541
+   layout = "float-right-desktop -background -shadow"
+%}
+
+Unfortunately, it is common for users to overpay on-chain fees by mistake. Wallets should have mechanisms in place to avoid this from happening.
+
+If a user's transaction amount is low compared to the fee they would pay to broadcast, warn them. There are no standards regarding the threshold percentage to trigger such an alert. A good benchmark is to warn the user if their fee is 50% or more than the value of the transaction itself. You might choose a different threshold for your app. Procedures like this should be enough to ensure users do not overpay on-chain fees unintentionally.
+
+</div>
+
+#### Including the fee in the amount
+
+<div class="center" markdown="1">
+
+{% include picture.html
+   image = "/assets/images/guide/daily-spending-wallet/sending/send-fees/fee-options-receiver-pays.png"
+   retina = "/assets/images/guide/daily-spending-wallet/sending/send-fees/fee-options-receiver-pays@2x.png"
+   modalImage = "/assets/images/guide/daily-spending-wallet/sending/send-fees/fee-options-receiver-pays-big.png"
+   alt-text = ""
+   caption = "Users can choose to put the fee burden on the receiver."
+   width = 250
+   height = 541
+   layout = "float-right-desktop -background -shadow"
+%}
+
+The most common scenario is that sender and receiver have an understanding of how much to exchange, and the sender is also responsible for paying the fee. However, there are situations where this is not wanted or possible:
+
+1. Sender and receiver agree that the receiver will cover the fee
+2. The sender balance is enough to cover the amount to pay, but not the fee
+3. The sender wants to clear out the wallet, possibly to migrate
+
+To accomodate these, the interface also needs to provide manual customization options, although they can be placed into secondary location.
```

Comment:
```suggestion
To accomodate these, the interface also needs to provide manual customization options, although they can be placed into secondary location.
```
It may help to elaborate on the "secondary location" mentioned here as a reader I am a bit unsure as to what that is.

Reply:
I added `(see below)`, since the next paragraph shows customization in detail.